src/osgAnimation/StatsHandler.cpp

```diff
@@ -440,14 +438,11 @@ struct ValueTextDrawCallback : public virtual osg::Drawable::DrawCallback
                     //_actions[name].touch();
                 }
                 _actions[name]._group->setNodeMask(~osg::Node::NodeMask(0x0));
-                size[name] = 0;
                 pos.y() -= characterSize + graphSpacing;
             }
 
             pos.y() -= backgroundMargin;
             osg::Vec3Array* array = static_cast<osg::Vec3Array*>(_background->getVertexArray());
-            float y = (*array)[0][1];
-            y = y - (pos.y() + backgroundMargin); //(2 * backgroundMargin + (size.size() * (characterSize + graphSpacing)));
```

Comment:
This and the four other lines above look like code that had been experimented with and then left unused.  Assigning the same pos.y value later in the code to multiple vertices looks a bit suspicious so I'm not confident that there isn't a wider bug here and it may be appropriate to re-instate these unused variables at somepoint.  Given this it's not appropriate to remove the code so I have commented it out and checked this into master.

Reply:
makes sense